seatunnel-connectors-v2/connector-cdc/connector-cdc-mysql/src/main/java/org/apache/seatunnel/connectors/seatunnel/cdc/mysql/config/MySqlSourceConfigFactory.java

```diff
@@ -76,7 +76,7 @@ public MySqlSourceConfig create(int subtaskId) {
         // Note: the includeSchemaChanges parameter is used to control emitting the schema record,
         // only DataStream API program need to emit the schema record, the Table API need not
```

Comment:
My suggestion is to add a comment explaining the reason for keeping "include.schema.changes" as false.

Reply:
Ok